sdk/tests/program-manager.test.ts

```diff
@@ -7,11 +8,13 @@ import {
     statePathRecordOwnerPrivateKey,
     stateRoot
 } from "./data/account-data";
-import { Account, ExecutionResponse, OfflineQuery, ProgramManager, RecordPlaintext } from "../src/node";
+import { IMPORT_1, IMPORT_2, MINT_VERIFYING_KEY, PROGRAM, SPEND_VERIFYING_KEY, SPIN_VERIFYING_KEY } from "./data/program";
+import { expect } from "chai";
 
 describe('Program Manager', () => {
     const programManager = new ProgramManager("https://api.explorer.provable.com/v1", undefined, undefined);
     programManager.setAccount(new Account({privateKey: statePathRecordOwnerPrivateKey}));
+    const network = (<AleoNetworkClient>programManager.networkClient).host  === "https://api.explorer.provable.com/v1/testnet" ? "testnet" : "mainnet";
```

Comment:
Should we add a property to make it easier to detect which network is being used?